src/SignUpIn/ReceivingAgency/ReceivingAgencySignUp3.js

```diff
@@ -0,0 +1,217 @@
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom';
+import truck from "../../icons/truck.svg";
+
+class ReceivingAgencySignUp3 extends Component {
+    constructor(props) {
+        super(props);
+        this.nextStep = this.nextStep.bind(this);
+    }
+
+    render() {
+        return (
+            <div className="signup-content">
+                <div className="form-block">
+                    <label className="form-component delivery">Delivery Days</label>
+                    <div className="row"> <input className="checkbox" type="checkbox" /><div className="day">Mon</div>
```

Comment:
maybe consider refactoring these options out into a small module or function instead of repeating it 7 times? Consider creating a list or a map for the 7 days and loop over the list/map. Feel free to msg me if you want to talk over this.

Reply:
You're super right that's a good idea. Won't have time to implement that before our meeting most likely but I will work on that.